### ISSUE TITLE
Bq24075 bat charge on h1

### ISSUE THREAD

**Issue by antonlalala** [sharebrained/portapack-hackrf#153]
Hello friends, are there people who soldered bq24075? On h1 
![9D9AF3F7-4543-42D3-A459-07D535A35B84](https://user-images.githubusercontent.com/43932931/77134885-163ef300-6a7a-11ea-99e5-68b0ea24980a.jpeg)
![5092FA25-B178-446D-A0A9-4C490011F719](https://user-images.githubusercontent.com/43932931/77134887-18a14d00-6a7a-11ea-972f-6452ea10332a.jpeg) 
How will it even turn on? by pressing which button? I did not understand.

**Comment by jboone** [comment]
Hi. Thank you for opening the issue here.

The news is not good. The battery feature was an experiment, and was never turned into a finished feature. So there is no way to turn the power on or off from the firmware.

**Comment by antonlalala** [comment]
☹️😢

**Comment by antonlalala** [comment]
GPS module for what purpose?

**Comment by jboone** [comment]
Also an experiment. There is no firmware support for it, and I have not tested it at all.

**Comment by antonlalala** [comment]
It would be nice to do GPS. Ais transponder, aprs ..

**Comment by jboone** [comment]
True. There's probably also value to recording current location when logging data, and having precise time available.

**Comment by antonlalala** [comment]
that would be great, as well as navigation

**Comment by antonlalala** [comment]
going back to bq. If I solder it according to the scheme, then the device will be constantly on while the battery is charged? or will I ruin the device?

**Comment by vasja2000** [comment]
hi! Colleagues, help me find the scheme exactly Portapack_h2. We need it.

**Comment by jboone** [comment]
@vasja2000 You need to contact the people who make the H2. I have no information on the H2. It is a completely different design, as far as I can tell.

**Comment by jboone** [comment]
@antonlalala Yes, the device will be on constantly, until the battery is discharged, or you plug in power to USB.

**Comment by eried** [comment]
BTW the on off works perfectly on the H2, such a shame that the pcb is not available anywhere... (I wish there was a way to have battery level on the UI)

**Comment by zhaohuansi** [comment]
The Chinese foundry is already produced. Give you a photo tomorrow.

**Comment by cql1983** [comment]
> 
> 
> BTW the on off works perfectly on the H2, such a shame that the pcb is not available anywhere... (I wish there was a way to have battery level on the UI)

H2 doesn't exist. It's just a modified version based on H1 that a hobbyist added logic chips for switches.

**Comment by eried** [comment]
> H2 doesn't exist.

On that ethereal non existant H2, on off works perfectly

**Comment by cql1983** [comment]
Yes, you can visit :  https://market.m.taobao.com/app/idleFish-F2e/widle-taobao-rax/page-detail?wh_weex=true&wx_navbar_transparent=true&id=596112633971&ut_sk=1.WRF5Cr3ABR8DAMOzG71j1M%2Bu_12431167_1588928139854.Copy.detail.596112633971.139574515&forceFlush=1 
H2 is Chinese fans modify.

**Comment by antonlalala** [comment]
> Привет. Спасибо за открытие вопроса здесь.
> 
> Новости не хорошие. Функция батареи была экспериментом и никогда не превращалась в готовую функцию. Так что нет возможности включить или выключить питание из прошивки.


can i put any dcdc converter between audio svdd and vbus? and switch to lipo?

**Comment by zjywlive** [comment]
Hello everyone, I'm sorry this is a message translated by computer,I am familiar with this designer, and I am also involved in the design of some peripheral parts.

**Comment by antonlalala** [comment]
No!
